src/session.ts

```diff
@@ -38,12 +37,25 @@ import {
   Metadata,
   MetadataCallback,
 } from '@google-cloud/common';
-import {CreateSessionOptions} from './common';
+import {google as spannerClient} from '../proto/spanner';
 
 export type GetSessionResponse = [Session, r.Response];
 
 export interface CreateSessionCallback {
-  (err: Error | null, session?: Session | null, apiResponse?: r.Response): void;
+  (
+    err: Error | null,
+    session: Session | null,
```

Comment:
This and `apiResponse` should be optional. Maybe consider using your `ResourceCallback` generic here instead.